tools/cmd.go

```diff
@@ -67,3 +70,33 @@ func mustGetDuration(cmd *cobra.Command, flagName string) time.Duration {
 	}
 	return val
 }
+
+func ResolveManifestFile(input string) (manifestName string, err error) {
+	if input == "" {
+		_, err := os.Stat("substreams.yaml")
+		if err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				return "", fmt.Errorf("no manifest entered in dir w/o a manifest")
+			}
+			return "", fmt.Errorf("finding manifest: %w", err)
+		}
+
+		return "substreams.yaml", nil
+	}
+
+	inputInfo, err := os.Stat(input)
+	if err != nil {
+		return "", fmt.Errorf("reading input: %w", err)
```

Comment:
> reading input

Feels like we are trying to read some content, maybe `read input file info: %w`.